ppx/browser/dune

```diff
@@ -0,0 +1,48 @@
+(library
+ (public_name melange-json.browser)
+ (name ppx_deriving_json_browser)
+ (modules
+  :standard
+  \
+  ppx_deriving_json_runtime
+  ppx_deriving_json_browser_test)
+ (libraries ppxlib containers)
+ (ppx_runtime_libraries melange-json.browser_runtime)
+ (preprocess
+  (pps ppxlib.metaquot))
+ (kind ppx_deriver))
+
+(library
+ (public_name melange-json.browser_runtime)
+ (name ppx_deriving_json_browser_runtime)
+ (modules ppx_deriving_json_runtime)
+ (wrapped false)
+ (modes melange))
+
+(executable
```

Comment:
could we use the `test` stanza here?

Reply:
not really, this thing makes a standalone driver, was useful for quick ad-hoc tests but feel free to remove it